### ISSUE TITLE
Public API for reader apps?

### ISSUE THREAD

**Issue by beville** [mcpierce/comixed#191]
I just played with this a bit, and I'm very impressed with what you've got so far.  I've got a collection tagged with ComicInfo.xml files, and it's great that it pulled in all the metadata, and can organize comics using it.  I really like the page repository idea for identifying and removing ads/scanner pager etc.

I'm curious: do you plan to publish a public API for 3rd-party reader apps (mobile or desktop) to access the library?   Something like OPDS, or what [ComicStreamer](https://github.com/beville/ComicStreamer) tried to do? Maybe this is already in the works?  Because for mobile especially, I've not seen any browser-based reader experience that can beat a dedicated app.

**Comment by mcpierce** [comment]
Yes. We've had one person give us a partial solution using OPDS. It needs to be reviewed, and fixed up in a few places, but it appeared to work well.
